eng/pipelines/templates/stages/archetype-spec-gen-sdk.yml

```diff
@@ -202,34 +212,34 @@ stages:
 
           if [ "$(Build.Reason)" != "PullRequest" ] && [ "${{ parameters.ConfigType }}" = "TypeSpec" ]; then
             optional_params="$optional_params --tsp-config-relative-path ${{ parameters.ConfigPath }}"
-            sdk_gen_info="$sdk_gen_info '${{ parameters.ConfigPath }}',"
+            sdk_gen_info="$sdk_gen_info '${{ parameters.ConfigPath }}',`n"
```

Comment:
"`n" doesn't work in 'bash' and will be printed literally in pipeline log, while it works for powershell.